routes/modelIndex.js

```diff
@@ -79,6 +78,41 @@ var route = function (req, res, next) {
             plural: req.linz.model.linz.formtools.model.plural
         },
         modelQuery: JSON.stringify(req.linz.model.formData)
+    };
+
+    const renderRecordAction = data.model.list.recordActions.renderer || recordActionRenderers.defaultRenderer;
+
+    data.records.forEach((record, index) => {
+
+        renderRecordAction({
```

Comment:
@allanchau, will it? Why don't you time it before and after the change... Just remember, you might need to run with `NODE_ENV=production` or something like that. Many Node.js renderers operate different based on that ENV variable, for example, some don't use template caching while in DEBUG mode.